### PR TITLE
add sepolicy rules for sensors user build

### PR DESCRIPTION
**Pull request by RajaniRanjan**
CTS-verifier Device suspend activity test case for user build.

Tracked-On: OAM-99476
Signed-off-by: RajaniRanjan <rajani.ranjan@intel.com>

### CODE REVIEW

**Comment by balajim001** [comment]
Please provide patch description

**Review by balajim001 — APPROVED** [review]
LGTM

**Review by swaroopbalan — APPROVED** [review]

**Comment by swaroopbalan** [comment]
approved to prepare pre intg , please also get it approved by zhenlong

File: sensors/mediation/sensor_hal_default.te [hunk 1]
```diff
@@ -2,9 +2,12 @@ allow hal_sensors_default self:socket create_socket_perms;
...
-    dontaudit hal_sensors_default default_prop:file { open read getattr map };
-    dontaudit hal_sensors_default port:tcp_socket { name_connect };
-    dontaudit hal_sensors_default self:tcp_socket { create read write connect name_connect getopt setopt };
-')
+allow hal_sensors_default self:tcp_socket { create read write connect name_connect getopt setopt };
+dontaudit hal_sensors_default default_prop:file { open read getattr map };
+allow hal_sensors_default port:tcp_socket { name_connect };
+
+#=================hal sensors_default=================
+allow hal_sensors_default vendor_intel_ipaddr_prop:file read;
```
Comment (ceiba1985) [inline sensors/mediation/sensor_hal_default.te]: use get_prop(hal_sensors_default, vendor_intel_ipaddr_prop) instead of line10 ~ line13

**Review by ceiba1985 — APPROVED** [review]

**Pull request merged**